pkg/scraper/jobcenter_scraper.go

```diff
@@ -10,59 +9,60 @@ import (
 )
 
 const (
-	pageSize = 30
+	pageSize = 200
 
 	jobcenterUrl = "https://www.jobcentrebrunei.gov.bn"
 )
 
 func ScrapeJobcenter() ([]*model.Job, error) {
-
 	jobs := []*model.Job{}
 
-	url := fmt.Sprintf("%s/web/guest/search-job?q=&delta=%d", jobcenterUrl, pageSize)
-
-	doc, err := getDocument(url)
-	if err != nil {
-		return nil, err
-	}
-
-	doc.Find("li.list-group-item.list-group-item-flex").EachWithBreak(func(i int, s *goquery.Selection) bool {
-
-		jobTitle := s.Find(".jp_job_post_right_cont h4 a").Text()
-		company := s.Find(".jp_job_post_right_cont p a").Text()
-		salary := s.Find(".jp_job_post_right_cont>ul li:first-child").Text()
-		location := s.Find(".jp_job_post_right_cont>ul li:nth-child(2)").Text()
-
-		link, exist := s.Find(".jp_job_post_right_cont h4 a").Attr("href")
-		if !exist {
-			return true
-		}
-
-		providerJobId, err := getJobcenterJobId(link)
-		if err != nil {
-			return true
-		}
+	for i := 1; i < 20; i++ {
+		url := fmt.Sprintf("%s/web/guest/search-job?q=&delta=%d&start=%d", jobcenterUrl, pageSize, i)
 
-		description, err := scrapeJobDescription(link)
+		doc, err := getDocument(url)
 		if err != nil {
-			return true
-		}
-
-		job := model.Job{
-			Provider:      JobCenter,
-			ProviderJobId: providerJobId,
-			Title:         jobTitle,
-			Company:       company,
-			Salary:        salary,
-			Location:      location,
-			Link:          link,
-			Description:   *description,
+			return jobs, nil
```

Comment:
I don't think this is the best way to do it.

For jobcentre I think even if you exceed the max page, it will just keep returning the data from the final page and this program will keep running for those extra pages for no reason.

Try to see if there is a way to get the total numbers of item or pages first, then loop through those. I think you should be able to get the final page from the page selector towards the bottom of the page.